db/migrate/20161122035953_question_order_type.rb

```diff
@@ -0,0 +1,5 @@
+class QuestionOrderType < ActiveRecord::Migration
+  def change
+    add_column :question_orders, :type, :string, null: false
```

Comment:
Type is what tells rails that we're using single table inheritance (i.e. DefaultQuestionOrder and ConditionalQuestionOrder being written to same table). It lets rails know which model to instantiate when it reads the row back from the db. This is more of a piece that we left out when building the question order models.